thrift-cluster/src/main/thrift/cluster.thrift

```diff
@@ -336,7 +338,9 @@ service RaftService {
   * When a follower finds that it already has a file in a snapshot locally, it calls this
   * interface to notify the leader to remove the associated hardlink.
   **/
-  void removeHardLink(1: string hardLinkPath)
+  void removeHardLink(1:string hardLinkPath)
+
+  void RefreshConnection(1:RefreshReuqest request)
```

Comment:
hi , I have a little question,  what does  RefreshConnection() do?    which is an empty implementation in MetaClusterServer and DataClusterServer.   if have a use, we can use refreshConnection. generate, Method Name Camel Command Format.

Reply:
For the `refreshConnection` rpc, It's just a simple implementation of application layer heartbeat preservation mechanism for long TCP connections.  For the detailed reason, you can refer to [jira1487](https://issues.apache.org/jira/browse/IOTDB-1487).

For the Camel Command Format, It's my fault and I will fix this. thanks for your reminding!

Reply:
Thanks. I get it.